styles/styles.css

```diff
@@ -378,19 +377,15 @@ main img {
 /* Image variations */
 .leadership-content img {
     border-radius: 15px;
-    }
-
-main .quarter-circle img {
-    clip-path: circle(100% at 100% 100%);
```

Comment:
How do images get the quarter circle clip path applied now? Or is up to the images to already be cropped?

Reply:
This code was cropping all images, whether or not they were right-facing or left-facing. We have right/left clipping happening on other classes, I was going to look around for any images that are not addressed after this.
![image](https://user-images.githubusercontent.com/3883795/230379466-9ae12cb1-fd53-471b-aa12-1f02552e3cb3.png)